.github/workflows/ruby.yml

```diff
@@ -1,77 +1,27 @@
 name: Ruby tests
+
 on:
   pull_request:
+
 env:
   BUNDLE_WITHOUT: journald:development:console:libvirt
   RAILS_ENV: test
   DATABASE_URL: postgresql://postgres:@localhost/test
   DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
```

Comment:
We don't have these in our common action. Are they still needed?

Reply:
No, we don't have to. I just looked into the foreman plugin test.